monai/networks/utils.py

```diff
@@ -264,8 +265,8 @@ def to_norm_affine(
         raise ValueError(f"affine suggests {sr}D, got src={len(src_size)}D, dst={len(dst_size)}D.")
 
     src_xform = normalize_transform(src_size, affine.device, affine.dtype, align_corners, zero_centered)
-    dst_xform = normalize_transform(dst_size, affine.device, affine.dtype, align_corners, zero_centered)
-    return src_xform @ affine @ torch.inverse(dst_xform)
+    dst_xform = normalize_transform(dst_size, "cpu", affine.dtype, align_corners, zero_centered)
+    return src_xform @ affine @ convert_to_dst_type(np.linalg.inv(dst_xform.numpy()), dst=affine)[0]
```

Comment:
Could you please add a comment here to explain why we convert to numpy for future understanding.

Thanks.

Reply:
I think @mingxin-zheng can create a ticket on github for the technical details, with similar content provided on nvbugs?

Reply:
Okay! Here it is:
https://github.com/Project-MONAI/MONAI/issues/5983

Reply:
Thanks, I've linked the pr to the ticket.

Reply:
Could you please just add 1 line comment to help others understand why we convert it to numpy array here?

Thanks.

Reply:
let me put the information on the conntected ticket.